src/Core/src/Handlers/SearchBar/SearchBarHandler.iOS.cs

```diff
@@ -37,9 +37,13 @@ protected override void ConnectHandler(MauiSearchBar platformView)
 
 			platformView.OnEditingStarted += OnEditingStarted;
 			platformView.OnEditingStopped += OnEditingEnded;
+				
+			if (_editor != null)
+				_editor.EditingChanged += OnEditingChanged;
+			
+			SetupDefaults(platformView);
```

Comment:
Can you remove the `SetupDefaults` from this? if you merge with main that might delete it.

Reply:
@PureWeen Updated, could you review it again?. Thanks!.